src/containers/TemplateBuilder/shared/OperationContextHelpers.tsx

```diff
@@ -238,11 +239,16 @@ export const importTemplate: ImportTemplateHelper =
 
       if (!result) return false
 
-      return await safeFetch(
+      const snapshotResult = await safeFetch(
         `${useSnapshotUrl}?name=${snapshotName}&optionsName=${templateExportOptionName}`,
         '{}',
         setErrorAndLoadingState
       )
+
+      // Delete the snapshot cos we don't want snapshots page cluttered with individual templates
+      safeFetch(`${deleteSnapshotUrl}?name=${snapshotName}`, {}, () => {})
```

Comment:
Don't need to await this, since the result doesn't affect anything on the current page.